@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp

```diff
@@ -9,6 +9,14 @@ class CfgFunctions
     class fza_ah64_project_sfmplus
     {
         tag = "FZA_sfmplus";
+        class actuator {
```

Comment:
Not related to this pr

Reply:
Agreed, these need to be removed as these functions do not exist in this PR

Reply:
Fixed